app/src/main/java/ar/com/wolox/android/training/ui/home/HomePresenter.java

```diff
@@ -0,0 +1,16 @@
+package ar.com.wolox.android.training.ui.home;
+
+import javax.inject.Inject;
+
+import ar.com.wolox.wolmo.core.presenter.BasePresenter;
+
+/**
+ * My <b>HomePresenter</b>.
```

Comment:
This is not necessary

Reply:
I put that comment because otherwise its rejects me because checkstyle (required a Javadoc commentary) 🤔